students/srepking/Lesson09/donors.py

```diff
@@ -0,0 +1,140 @@
+
+
+class Group:
+
+    def __init__(self):
+
+        self._donor_raw = {}
+
+    def get_donors(self):
+        return self._donor_raw
+
+    donors = property(get_donors)
+
+    def search(self, donor):
+        if self._donor_raw.get(donor) is None:
+            return
+        else:
+            return donor
+
+    def add(self, donor, donation):
+        if self.search(donor) is None:
+            self._donor_raw[donor] = [donation]
+        else:
+            self._donor_raw[donor].append(donation)
+
+    def create_list(self):
```

Comment:
more appropriate name would be -> `def print_donors(self):`

Reply:
Updated my code.

Reply:
I made the change.